src/redux/app/saga.js

```diff
@@ -371,7 +372,10 @@ function* initFcmRequest() {
 function* getServerInfoRequest() {
   // 2. Test server connection and get Server info
   try {
-    const response = yield call(ParseHelper.getServerInfo);
+    const state = yield select();
+    const osType = Platform.OS === 'ios' ? 'iOS' : 'Android';
```

Comment:
better to define those osType values as constants, the same as rwallet-server does